microsoft-edge/devtools-guide-chromium/whats-new/2023/04/devtools-112.md

```diff
@@ -0,0 +1,176 @@
+---
+title: What's new in DevTools (Microsoft Edge 112)
+description: "The Performance tool can now unminify JavaScript running in out-of-process iframes, the CSS Overview tool displays a list of non-simple selectors for quick performance wins, and more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 04/08/2023
+---
+# What's New in DevTools (Microsoft Edge 112)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Performance tool unminifies JavaScript running in out-of-process iframes
+<!-- ## The Performance tool can now unminify JavaScript running in out-of-process iframes -->
```

Comment:
@zoherghadyali - use this new, telegraphic (shortened) heading, or the original heading that's hidden in a comment?